docs/tutorial/getting-started.md

```diff
@@ -1,3 +1,215 @@
 # Getting started
```

Comment:
The heading should be more specific, "Getting started with Pebble".

In the current Pebble doc there is a single tutorial, but if / when the product becomes more complex you may want to create a second tutorial that focuses on a different learning journey. So headings should be descriptive.

For reference:
https://juju.is/docs/juju/tutorial
https://documentation.ubuntu.com/lxd/en/latest/tutorial/first_steps/
https://maas.io/docs/tutorial-bootstrapping-maas

Reply:
The doc site has and only has Pebble, unlike the Juju docs site where you find both Juju and Juju SDK.

I feel like the "with pebble" part is redundant. Pushing back gently.